packages/fuzzer/utils.cpp

```diff
@@ -19,6 +19,16 @@
 
 #define btoa(x) ((x) ? "true" : "false")
 
+bool sigintDetected = false;
+
+bool getSigintCalled() { return sigintDetected; }
+
+void sigintHandler(int signum) {
```

Comment:
Could you add a comment describing why the signal handler callback saved in `fuzzing_sync` is not executed directly? As far as I understand the code, the flag would still be needed to block further fuzz target executions during shutdown.

Reply:
It was some weird node "V8" "wrong context"(? not sure about this one) ... something error when that function was called from the handler directly. So I moved the call to the function that already calls a node function without problems. Could it be that "env" variable that's missing?

Reply:
Added a comment.